hack/featuregate_backup.yaml

```
@@ -0,0 +1,14 @@
+---
+apiVersion: config.openshift.io/v1
+kind: FeatureGate
+metadata:
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/single-node-developer: "true"
+    release.openshift.io/create-only: "true"
+  name: cluster
+spec:
+  customNoUpgrade:
+    enabled:
+      - AutomatedEtcdBackup
```

Sorry for the nit but the feature gate name is `AutomatedEtcdBackupAPI`. 
```suggestion
      - AutomatedEtcdBackupAPI
```
https://github.com/openshift/api/blob/master/config/v1/feature_gates.go#L285

---

Nvm, this was fixed in https://github.com/openshift/api/pull/1518